README.md

```diff
@@ -76,9 +76,9 @@ The installer will also set Apache group ownership of certain /data/ subdirector
 
   To help ensure Apache-writability, verify that your environment file contains the following line:
 
-    ```bash
-    umask 0002
-    ```
+   ```bash
+   umask 0002
+   ```
 
```

Comment:
It's currently looking a bit weird:
https://github.com/aces/Loris-MRI/blob/f2136465bfcc7902105b93fdd9db9cd295696cd5/README.md

If merged:
https://github.com/Jkat/Loris-MRI/blob/ad0e387a8056d3001119ae64900a6a74c309671c/README.md